src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.Button;
+
+/**
+ * A wrapper class for our custom-made operator board. Does not have a
+ * constructor. Access statically. Access Buttons via public static state.
+ * Access axis values via methods. Use methods to make buttons light up.
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
```

Comment:
Why are these not static and final?

Reply:
These need to be accessed through an instance so that I can ensure that these are constructed, and that the OperatorBoard constructor is run.  And I ensure that this is a singleton class with the .getInstance method and having a private OperatorBoard constructor.

Reply:
if there not static and final then why are you using the constant naming convention?

Reply:
Can't they be constructed inline? Why do you need a constructor at all?